src/pages/createIndividual/CreateIndividual.jsx

```diff
@@ -45,7 +45,7 @@ export default function CreateIndividual() {
   useEffect(() => {
     const initialState = calculateInitialState(createFieldSchemas);
     setFormState(initialState);
-  }, [createFieldSchemas?.length]);
+  }, [createFieldSchemas, createFieldSchemas?.length]);
```

Comment:
You don't need the `createFieldSchemas?.length` as a dependency. `createFieldSchemas` is redeclared on every render, so this useEffect will run on every render.

Reply:
i actually modified `createFieldSchemas` to be inside a `useMemo`. otherwise there was an infinite loop. but i am gonna switch the length check from createFieldSchemas in the useEffect to fieldSchemas on useMemo